packages/react-native-otlp/src/index.ts

```diff
@@ -74,9 +74,8 @@ const initialize = (otlpExporterConfig: OTLPExporterConfig) => {
         sdkConfig,
         otlpExporterConfig,
       );
-    } catch (error) {
-      console.warn(WARN_MESSAGES.error, error);
-      return Promise.reject(false);
+    } catch (_) {
+      return Promise.reject(WARN_MESSAGES.error);
```

Comment:
@jpmunz I've updated this block since I guess it makes sense to reject with at least an error message instead of a boolean value. I think there is no scenario when the code will throw (since it is always resolving the promise with the boolean) but I guess it makes sense to keep the try/catch due to possible exceptions coming from the react native layer